mod/05_email/set/type/email_template.rb

```diff
@@ -44,7 +44,7 @@ def email_config args={}
   
   [:subject, :text_message].each do |field_name|
     config[field_name] = process_email_field( field_name, args ) do |field_card|
-      field_card.contextual_content context_card, {:format=>'email_text'}, args
+      field_card.contextual_content context_card, {:format=>'email_text'}, args.merge(:content_opts=>{ :chunk_list=>:inclusion_and_link})
```

Comment:
Look at content_specs:
116 TEXT_RENDERED = {
117   :three => [
118     "Some URIs and Links: ", 'http://a.url.com/',
119     "\n        More urls: ",
120     "wagn.com/a/path/to.html[http://wagn.com/a/path/to.html]",
121     "\n        ",
122     "http://localhost:2020/path?cgi=foo&bar=baz",
123     "  ",
124     "extra[http://brain.org/Home]",
125     "\n        [ ",
126     "http://gerry.wagn.com/a/path",
127     " ]\n        { ",
128     "https://brain.org/more?args",
129     " }"
130   ],
131 }
You can look at the input too, there is one case where it does a free link with the [] added on, but that's because the link and the text are not the same.  The link version has http: added because it is a host without protocol url.

We need a clearer spec for this, and if there is special handling for subject, maybe it is to pass an additional option arg to tell the link formatter to just render the original text (which is what this will do, right?).